Editor/IO/Files/CopyFilesToDirectory.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace UniTools.Build
+{
+    [CreateAssetMenu(
+        fileName = nameof(CopyFilesToDirectory),
+        menuName = MenuPaths.IO + nameof(CopyFilesToDirectory)
+    )]
+    public sealed class CopyFilesToDirectory : BuildStep
+    {
+        [SerializeField] private PathProperty[] m_filePaths = default;
+        [System.Obsolete, HideInInspector]
+        [SerializeField] private PathProperty m_filePath = default;
```

Comment:
Looks like can be removed

Reply:
Agreed. Since we won't be replacing CopyFileToDirectory, we don't need to replicate its properties.